### ISSUE TITLE
Reduce number of coverage comments on PRs to one

### ISSUE THREAD

**Issue by andy-maier** [zhmcclient/python-zhmcclient#173]
Currently, the coveralls service posts a number of comments for the initial PR and for each update of a PR. All these comments from one "run" provide the same link.

The number of such comments for the same "run" and link is the number of successful Travis jobs. Starting with PR #172, the Travis CI runs for Python 2.7, 3.4, 3.5, 3.6, and pypy. The Travis control file invokes coveralls at the end upon success. In all five cases, the Travis run succeeds up to the point where coveralls runs. For the python x.y versions, coveralls itself also succeeds, each one resulting in a comment on the PR. For pypy, coveralls is attempted to be run, but fails with an exception, so no coverage comment is posted for that environment.

Some investigation on the coveralls.io links: The link in the little badge in each coveralls comment targets a "build" of Travis (e.g. https://coveralls.io/builds/9852638 for Travis build 820). A "build" is the set of all Travis Python environments in a particular PR. The run for each Travis Python environment within a PR is termed a "job". There are also links to coveralls.io pages for those jobs (e.g. https://coveralls.io/jobs/22316159 for Travis job 820.1 within Travis build 820). The coveralls.io page for the build also has the links for the job pages.

Bottom line, it would be sufficient to have exactly one posted coveralls comment with the link it currently has (to the page for the "build"), and not four of them.

This issue is to reduce the number of coverage comments that are posted by the coveralls service for a particular Travis build of a PR, to one.

**Comment by andy-maier** [comment]
It turns out that other projects have this issue, too. The coveralls project currently has two issues open for that:
* lemurheavy/coveralls-public#491
* lemurheavy/coveralls-public#512
Another issue on this problem was open and has been closed without resolving it, although it mentions a fix would be in place:
* lemurheavy/coveralls-public#838

A number of people that had this issue recommend moving from coveralls to codecov, e.g. nvbn/thefuck#566.

**Comment by andy-maier** [comment]
PR #174 mitigates the problem. The downside of that PR is that coverage is being measured only for Python 2.7 now. May be good enough for now.

**Comment by andy-maier** [comment]
Discussion point: Are we ok with this fix as a permanent fix? The downside is that we get coverage results only for Python 2.7 now.

**Comment by andy-maier** [comment]
Will close this one by end of next week if nobody objects.

**Comment by andy-maier** [comment]
Closing this issue as nobody has objected.
